ARMmbed/mbed-os
# Fixing a macro to detect when RTOS was in use for the NRF52840_DK

## Description
The recently added NRF52840_DK broke the mbed 2 nightly. This is because the check for the presence of the RTOS was using the wrong macro. I have fixed it in this PR.


## Status
**READY**


## Migrations
If this PR changes any APIs or behaviors, give a short description of what *API users* should do when this PR is merged.

NO


## Todos
- [x] morph test
- [x] export build
- [x] mbed 2 bot


## Steps to test or reproduce
Currently on master if you run the following command:

```
$ python tools/build_release.py -t GCC_ARM -p NRF52840_DK -v
```

You will get the following error:

```
[Error] app_timer_rtx.c@47,0:  #5: cannot open source input file "cmsis_os.h": No such file or directory
```

This branch should fix the issue.
